ci/travis/install-dependencies.sh

```diff
@@ -24,19 +24,17 @@ if [[ "$PYTHON" == "2.7" ]] && [[ "$platform" == "linux" ]]; then
   wget https://repo.continuum.io/miniconda/Miniconda2-4.5.4-Linux-x86_64.sh -O miniconda.sh -nv
   bash miniconda.sh -b -p $HOME/miniconda
   export PATH="$HOME/miniconda/bin:$PATH"
-  pip install -q cython==0.29.0 gym==0.10.11 opencv-python-headless pyyaml pandas==0.23.4 requests \
+  pip install -q scipy tensorflow cython==0.29.0 cmake gym==0.10.11 opencv-python-headless pyyaml pandas==0.23.4 requests \
```

Comment:
we don't need cmake here (and below), right?

Reply:
yes, fixed